runtime/compiler/z/codegen/J9CodeGenerator.cpp

```diff
@@ -3743,7 +3749,13 @@ J9::Z::CodeGenerator::inlineDirectCall(
    // If the method to be called is marked as an inline method, see if it can
    // actually be generated inline.
    //
-   if (comp->getSymRefTab()->isNonHelper(node->getSymbolReference(), TR::SymbolReferenceTable::currentTimeMaxPrecisionSymbol))
+
+   if (comp->getSymRefTab()->isNonHelper(node->getSymbolReference(), TR::SymbolReferenceTable::encodeASCIISymbol) && cg->getSupportsInlineEncodeASCII())
```

Comment:
Thanks for pointing this out. I'd say right now this check isn't needed since the `InlineEncodeASCII` flag is initially set by the codegen itself and we only generate this IL in the JIT if the flag is set. So that makes this check during tree evaluation redundant. I've removed the check here: 18163e58a18208669debb6efd2261b8e744b8660. If you think we should be more cautious and add it back, then let me know. I think the best place to add it would be in the `inlineEncodeASCII` routine itself then.

Reply:
Simply removing the check is fine by me :+1: